github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -2589,3 +2589,175 @@ periodics:
         privileged: true
     nodeSelector:
       type: bare-metal-external
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: prow-workloads
+  cron: 10 3,7,15,23 * * *
```

Comment:
Do we need these lanes running 4 times a day? Its a extra set of periodics which will add load. I would be more in favor of one or two runs a day.